main.go

```diff
@@ -33,9 +36,13 @@ func main() {
 			err := client.PostComment(pr, ghComment)
 			if err != nil {
 				fmt.Printf("Failed to post comment: %v\n", err)
-				fmt.Printf("Comment: %v\n", ghComment)
+				// TODO post comment to file.
```

Comment:
Clarification: What do you mean by post comment to file?

Reply:
Thank you for your feedback! The AI gives your comment a score of 3. This code review comment could be more specific and give concrete examples of what "post comment to file" means.

Reply:
I'll clarify the comment, will be a fast follow.